{{cookiecutter.project_name}}/docker/django/gunicorn.sh

```diff
@@ -26,15 +26,13 @@ python /code/manage.py compilemessages
 
 # Start gunicorn:
 # Docs: http://docs.gunicorn.org/en/stable/settings.html
+# Concerning `workers` setting see:
+# https://github.com/wemake-services/wemake-django-template/issues/1022
 /usr/local/bin/gunicorn server.wsgi \
-  # Sync worker settings:
-  # https://github.com/wemake-services/wemake-django-template/issues/1022
-  --workers=4 \
+  --workers=4 `# Sync worker settings` \
```

Comment:
Just out of curiosity: do we need backticks here, or is it a style thing?

Reply:
[Absolutely needed](https://stackoverflow.com/questions/9522631/how-to-put-a-line-comment-for-a-multi-line-command#comment24611094_12797512).